### PR TITLE
Major refactor of getting-started (w/readfiles)

### PR DESCRIPTION
**Pull request by ghost**
Opening this to show how readfile would work instead of shortcode alone.

Published to site:
screen shot 2018-09-19 at 3 30 59 pm

### CODE REVIEW

**Comment by ghost** [comment]
Will close #3 in favor of this approach based on discussion with @dcwangmit01

File: config.toml [hunk 1]
```diff
@@ -42,14 +42,30 @@ googleAnalytics = ""
...
-        name   = "About KDF"
+        name   = "About"
         url    = "/"
         weight = 10
 
 [[menu.main]]
-        name   = "Quick Start"
-        url    = "quick-start/"
-        weight = 20
+        name       = "Getting Started"
```
Comment (dcwangmit01) [inline config.toml:50]: SHOULD: Whitespace/tab issues.

File: content/getting-started/mac/index.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+```
+{{% /note %}}
+
+## Step 5: KDK
+
+{{% note %}}
+{{% readfile file="/content/getting-started/readfiles/kdk-note.md" markdown="true" %}}
+{{% /note %}}
+
+{{% readfile file="/content/getting-started/readfiles/kdk-install.md" markdown="true" %}}
```
Comment (dcwangmit01) [inline content/getting-started/mac/index.md:84]: Kind of sucks that Hugo requires the full path here.

File: content/getting-started/windows/index.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+- `Windows Search` -> `Git Bash`
+
+## Step 3: Docker
+
+### Install [Docker for Windows](https://docs.docker.com/docker-for-windows/release-notes/).
+
+```bash
+choco install -y docker-for-windows
+
+choco install -y docker
```
Comment (dcwangmit01) [inline content/getting-started/windows/index.md]: Should be removed right?

**Review by dcwangmit01 — APPROVED** [review]
💯

File: content/getting-started/windows/index.md (hunk 3 above)
Comment (ghost) [reply content/getting-started/windows/index.md]: @dcwangmit01  open to your thoughts.  see: https://github.com/docker/docker.github.io/issues/6910#issuecomment-421193847

where person implies CLI is in a separate package.

File: content/getting-started/mac/index.md (hunk 2 above)
Comment (ghost) [reply content/getting-started/mac/index.md:84]: +1. 

Relative path didn't work for me, so dunno what to do 

```
{{% readfile file="../readfiles/post-install.md" markdown="true" %}}
```

File: config.toml (hunk 1 above)
Comment (ghost) [reply config.toml:50]: Done. Thanks!

File: content/getting-started/windows/index.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
+---
+title: Getting Started > Windows
+---
+
+## Requirements
+
+- **Windows 10 Pro** or **Windows 10 Enterprise** is required.
+- You must execute all PowerShell commands as an adminstrative user.
```
Comment (rtluckie) [inline content/getting-started/windows/index.md]: administrative

Comment (ghost) [reply content/getting-started/windows/index.md]: Fixed. Thanks!

File: content/getting-started/windows/index.md (hunk 3 above)
Comment (dcwangmit01) [reply content/getting-started/windows/index.md]: Second install is not necessary.  Just tested on a fresh Win10 box.

Comment (ghost) [reply content/getting-started/windows/index.md]: Thanks @dcwangmit01 . Pushed fix for that.

Will double check on the Mac side. Standby.

Comment (ghost) [reply content/getting-started/windows/index.md]: Second brew conflicts too. Will push fix in a moment.

```
 brew install docker
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 5 taps (homebrew/cask, homebrew/cask-fonts, homebrew/cask-versions, homebrew/core, johanhaleby/kubetail).
==> Updated Formulae
faas-cli ✔                         cnats                              homebank                           root                               sqlite-analyzer
johanhaleby/kubetail/kubetail ✔    cpmtools                           libphonenumber                     roswell                            webpack
sqlite ✔                           dbhash                             mongodb@3.6                        rustup-init                        xcodegen
angular-cli                        erlang@20                          ntopng                             skaffold
avfs                               gloox                              proj                               sops
bullet                             helmfile                           pulumi                             sqldiff

==> Downloading https://homebrew.bintray.com/bottles/docker-18.06.1.sierra.bottle.tar.gz
######################################################################## 100.0%
==> Pouring docker-18.06.1.sierra.bottle.tar.gz
Error: The `brew link` step did not complete successfully
The formula built, but is not symlinked into /usr/local
Could not symlink bin/docker
Target /usr/local/bin/docker
already exists. You may want to remove it:
  rm '/usr/local/bin/docker'

To force the link and overwrite all conflicting files:
  brew link --overwrite docker

To list all files that would be deleted:
  brew link --overwrite --dry-run docker

Possible conflicting files are:
/usr/local/bin/docker -> /Applications/Docker.app/Contents/Resources/bin/docker
==> Caveats
Bash completion has been installed to:
  /usr/local/etc/bash_completion.d

zsh completions have been installed to:
  /usr/local/share/zsh/site-functions
==> Summary
🍺  /usr/local/Cellar/docker/18.06.1: 8 files, 52.3MB
```

Comment (ghost) [reply content/getting-started/windows/index.md]: Pushed fix for mac.

**Comment by ghost** [comment]
Fixed all comments. Okay to merge?

**Pull request merged**